corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -552,6 +553,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             'keypress @ui.searchTextBox': 'searchTextKeyAction',
             'keypress @ui.paginationGoTextBox': 'paginationGoKeyAction',
             'keypress @ui.paginators': 'paginateKeyAction',
+            'click @ui.mapShowHideButton': 'showHideMap',
```

Comment:
nit: would be nice to have this organized with the rest of `click` actions

Reply:
moved